features/named-color.yml

```diff
@@ -2,6 +2,7 @@ name: Named colors
 description: Some CSS color values can be referenced by name, such as `red` or `limegreen`. They stand in for specific RGB color values.
 spec: https://drafts.csswg.org/css-color-4/#named-colors
 group: color-types
+caniuse: css-rebeccapurple
```

Comment:
There is not a caniuse entry for named colors. It's a valid question, though. For the main use case of caniuse adding a Baseline banner to `rebeccapurple`, it would not be confusing. But are there other potential uses of a caniuse key that this would break?

Reply:
Our docs say `Setting a `caniuse` value says that a feature is approximately equivalent to or a superset of a Can I Use feature.` Since named-colors contains rebeccapurple on web-features, this is okay. I'll open a separate PR with another set of similar situations I identified.

Reply:
Going back through my list, most of the superset features have significant discrepancies, so I ended up just adding caniuse for `display`, `events`, and `font-kerning` on this branch. Apologies for adding after the approval!